libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -203,6 +203,8 @@ const AP_Param::GroupInfo RC_Channel::var_info[] = {
     // @Values{Copter}: 99:AUTO RTL
     // @Values{Copter, Rover, Plane, Blimp}: 100:KillIMU1, 101:KillIMU2
     // @Values{Copter, Rover, Plane}: 102:Camera Mode Toggle
+    // @Values{Copter, Rover, Plane, Blimp}: 103: Trigger EKF lane switch attempt
+    // @Values{Copter, Rover, Plane, Blimp}: 104: Trigger EKF yaw reset
```

Comment:
What makes these not available on Sub and Tracker?

Reply:
Nothing, it used to be that they were not enumerated specifically anywhere so they would get all options. Although that seems not to be the case anymore, I think they will currently just get 112.

Reply:
Yeah, https://ardupilot.org/antennatracker/docs/parameters.html#rc1-option-rc-input-option. Removing the mark-up for 112 doesn't seem to fix tho. I guess there are lots of options that need that mark-up adding. When I first split the descriptions over multiple lines it would export all options in that case. 

Anyway, I will add them here too.